classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -99,14 +126,18 @@ abstract class ActionScheduler_Abstract_ListTable extends WP_List_Table {
 	 * validations and afterwards will execute the bulk method, with two arguments. The first argument
 	 * is the array with primary keys, the second argument is a string with a list of the primary keys,
 	 * escaped and ready to use (with `IN`).
+	 *
+	 * @var array
 	 */
 	protected $bulk_actions = array();
 
 	/**
 	 * Makes translation easier, it basically just wraps
 	 * `_x` with some default (the package name).
 	 *
-	 * @deprecated 3.0.0
```

Comment:
Was this dropped accidentally? Seems like we'd want to preserve it and I don't think it breaks our ruleset.

Reply:
Yes, my bad. It's back now, I've added the text `Use `__()` instead.` along with it, does it make sense to be there?

Reply:
The method description suggests it is a wrapper around `_x()` (even though it isn't) and, like `_x()` it accepts a `$context`, so _probably_ this would be just a little better:

```
* @deprecated 3.0.0 Use `_x()` instead.
```

Reply:
Makes sense, changed text added.